inttests/GOSCALEIO_TEST.env_example

```diff
@@ -3,13 +3,15 @@ GOSCALEIO_INSECURE=true
 GOSCALEIO_HTTP_TIMEOUT=60
 GOSCALEIO_ENDPOINT=https://1.2.3.4:443
 GOSCALEIO_USERNAME=admin
-GOSCALEIO_PASSWORD=Password123
+GOSCALEIO_PASSWORD=Password
```

Comment:
Why are we updating the dummy password here?

Reply:
it was mentioned actual password which we are generally using.

Reply:
Well some of the arrays do work with Password123, but since int-test are executed only after the env variables are set locally, so it wont matter updating this field with any new value.

Reply:
Alright